src-packed/suggestions.js

```diff
@@ -64,7 +64,16 @@ const allReplacements = {
     "grandfather": ["flagship", "established", "rollover", "carryover", "classic"],
     "legacy": ["flagship", "established", "rollover", "carryover", "classic"],
     // custom negative sentiment just leave replacements empty
+    // https://www.oxfordinternationalenglish.com/dictionary-of-british-slang/
     "is pants": [],
+    "naff": [],
+    "tosh": [],
+    "eejit": [],
+    "faff": [],
+    "gobby": [],
+    "numpty": [],
+    "shirty": [],
+    "wonky": [],
```

Comment:
Do any of these get underlined as negative?

Reply:
some are picked up by the negative sentiment but some are not. So I thought I we just put all of these in.